### DISCUSSION TITLE
Cucumber for Makefile

### DISCUSSION THREAD

**Discussion by FaiqueAli** [cucumber/common#2188] [Q&A C++]
Hi, I have gone through the documentation and tried with the example project to understand `cucumber` and executed the defined test cases.
Now, I need to integrate `Cucumber`  in my QNX c++ project that is based on `Makefile`, Is there any documentation or sample project available to start with or any other reference point?

**Comment by mpkorstanje** [comment]
Moved to https://github.com/cucumber/cucumber-cpp/issues/302
